pmu_el0_cycle_counter.c

```diff
@@ -214,11 +215,21 @@ pmuctl_ioctl(struct file *f, unsigned int cmd, unsigned long arg)
 		return -ENOTTY;
 
 	if (_IOC_DIR(cmd) & _IOC_READ)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 0, 0)
```

Comment:
Could you add compat.h to avoid #ifdef in everywhere?
Refernce from KNI:  kernel/linux/kni/compat.h